pkg/ccl/changefeedccl/changefeed_processors.go

```diff
@@ -2030,6 +2028,13 @@ func (cf *changeFrontier) managePerTableProtectedTimestamps(
 		updatedPerTablePTS = true
 	}
 
+	if len(tableIDsToCreate) > 0 {
```

Comment:
Seems like we could actually get this down from 2 job_info writes to 1 by doing both the releasing/creating before writing